### PR TITLE
Pond container and pipe stuff

### PR DESCRIPTION
**Pull request by P0NDER0SA**
just like it says....

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM node:11.13.0-alpine
+
+# create destination directory
+RUN mkdir -p /usr/src/poc-intake
```
Comment (andr0272) [inline Dockerfile:4]: WORKDIR will create the directory if it doesn't exist. This extra layer is unneeded.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+WORKDIR /usr/src/poc-intake
+
+# update and install dependency
+RUN apk update && apk upgrade
+
+COPY . /usr/src/poc-intake/
+RUN npm install
```
Comment (andr0272) [inline Dockerfile:11]: Your already in /urs/src/poc-intake so just COPY . .

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,25 @@
+FROM node:11.13.0-alpine
+
+# create destination directory
+RUN mkdir -p /usr/src/poc-intake
+WORKDIR /usr/src/poc-intake
+
+# update and install dependency
+RUN apk update && apk upgrade
+
+COPY . /usr/src/poc-intake/
```
Comment (andr0272) [inline Dockerfile:10]: Copy package.json and lockfile first so dependencies dont need to be reinstalled on each code change

Comment (P0NDER0SA) [reply Dockerfile:10]: all noted.... i'm rebuilding this file now

**Pull request merged**